tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoFixture.NUnit3" Version="4.15.0" />
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="NUnit" Version="3.13.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\NHSD.BuyingCatalogue.Ordering.Api.UnitTests\NHSD.BuyingCatalogue.Ordering.Api.UnitTests.csproj" />
+    <ProjectReference Include="..\NHSD.BuyingCatalogue.Ordering.Common.UnitTests\NHSD.BuyingCatalogue.Ordering.Common.UnitTests.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="AutoFixture\" />
+  </ItemGroup>
+
+</Project>
```

Comment:
We treat warning as errors in release builds so can you enable that for the release build? Also make sure the release build also has the element below.

```xml
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Reply:
Updated warning as error in release builds and added `GenerateDocumentationFile` element